spec/models/artist_spec.rb

```diff
@@ -17,6 +19,8 @@
   it { should respond_to(:birth_date) }
```

Comment:
Haha, I know how you feel. There are a lot of opinions on what you should/shouldn't test, but I like [this approach best](http://www.confreaks.com/videos/2735-wickedgoodruby-outside-in-testing) (or [this one](http://blog.testdouble.com/posts/2014-05-25-breaking-up-with-your-test-suite.html)).

Don't worry about this test for now :smile: